pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -311,20 +311,32 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
-func getDruidDeployCommands(config *config.GardenletConfiguration) []string {
+func getDruidDeployCommands(gardenletConf *config.GardenletConfiguration) []string {
 	command := []string{"" + "/bin/etcd-druid"}
 	command = append(command, "--enable-leader-election=true")
 	command = append(command, "--ignore-operation-annotation=false")
-	command = append(command, "--workers=50")
-	command = append(command, "--compaction-workers=3")
+	if gardenletConf == nil {
+		return command
+	}
+
+	config := gardenletConf.ETCDConfig
+	if config == nil {
+		return nil
+	}
+	if config.ETCDController != nil {
+		command = append(command, "--workers="+strconv.FormatInt(pointer.Int64Deref(config.ETCDController.Workers, 3), 10))
+	}
+
+	if config.CustodianController != nil {
+		command = append(command, "--custodian-workers="+strconv.FormatInt(pointer.Int64Deref(config.CustodianController.Workers, 3), 10))
```

Comment:
Considering that the default worker count for the etcd-controller will is `50`, I think we should increase this value as well.

Reply:
Should 10 be good ? or 50?